pkg/network/protocols/http/incomplete_stats.go

```diff
@@ -12,11 +12,17 @@ import (
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/network/protocols"
 	"github.com/DataDog/datadog-agent/pkg/network/types"
 )
 
 const defaultMinAge = 30 * time.Second
 
+type connectionKey struct {
```

Comment:
Why not move the whole `ConnectionKey` struct here?

Reply:
sorry I miss read your previous comment

Reply:
:+1: